test/ios/mocked_data/test_get_bgp_neighbors_detail/normal/show_ip_bgp_all_sum.txt

```diff
@@ -34,7 +34,8 @@ BGP using 39527928 total bytes of memory
 BGP activity 3436532/2710503 prefixes, 19215241/17575537 paths, scan interval 60 secs
 
 Neighbor        V           AS MsgRcvd MsgSent   TblVer  InQ OutQ Up/Down  State/PfxRcd
-2001:559::1     4         7922 6944479  138054 116434201    0    0 12w2d       45646

```

Comment:
To ensure your changes don't break the existing behaviour, please don't change this file and provide a separate test case with the output you're seeing.

Reply:
The `show_ip_bgp_all_sum.txt` is the base for the whole test. Following your suggestion I've added a new neighbor to the list and kept all the original neighbors in place. Please let me know if this makes sense to you?

Reply:
We usually prefer to avoid changing this file when fixing a bug (as probably you won't see both outputs, i.e., in both `12:22:20` and `12w2d` format at the same time -- but never say never Cisco is Cisco lol), and have a separate test case; that means, another set of files under a directory, e.g., ` test/ios/mocked_data/test_get_bgp_neighbors_detail/<some test case name>/`.
But this change is small enough we can leave this pass...

Reply:
All right, it all makes sense now. I wasn't quite sure how the test framework works. Thank you for the explanation, I'll keep it in mind for the next time!